app/handlers/signup_start.rb

```diff
@@ -51,6 +56,27 @@ def handle
     outputs.signup_state = new_signup_state
   end
 
+  def confirm_trusted_account(signup_state)
+    user = User.create
+    user.role = signup_state.role
+    user.full_name = trusted_state['name']
+    user.save
+
+    transfer_errors_from(user, {type: :verbatim}, true)
+    ci = user.contact_infos.build(
+      type: 'EmailAddress',
+      value: signup_state.contact_info_value
+    )
+    run(ConfirmContactInfo, ci)
+    transfer_errors_from(ci, {type: :verbatim}, true)
+    options[:session].sign_in! user
+
+    uuid = UserAlternativeUuid.create(user: user, uuid: trusted_state['uuid'])
+    transfer_errors_from(uuid, {type: :verbatim}, true)
+
+    outputs.redirect_action = :password
```

Comment:
Seems like maybe we want the calling app to indicate whether humans sent to accounts should be asked for a password or not... i.e. for LMS students Tutor can say "doesn't need a password", but for LMS teachers we would want a password so they can use their account for other OX uses.

Reply:
Actually i kinda wonder if we want to give students a password.  We'd decided not to, but that was when we planned to just auto-create an account for them.  Maybe it makes sense to do so now?